cranelift/codegen/src/lib.rs

```diff
@@ -17,10 +21,12 @@ extern crate alloc;
 #[macro_use]
 extern crate std;
 
+// Supports compiling ISLE.
 #[cfg(not(feature = "std"))]
+#[macro_use]
+extern crate core as std;
```

Comment:
Oh this shouldn't be done in my opinion, it can create a lot of confusion by renaming core crates to each other.

I'd recommend following the pattern of Wasmtime crates which is to do:

```rust
#[cfg(feature = "std")]
#[macro_use]
extern crate std;
```